src/v/cluster/config_manager.cc

```diff
@@ -345,8 +345,8 @@ static void preload_local(
   std::optional<std::reference_wrapper<config_manager::preload_result>>
     result) {
     auto& cfg = config::shard_local_cfg();
-    auto& property = cfg.get(key);
     if (cfg.contains(key)) {
+        auto& property = cfg.get(key);
```

Comment:
`cfg` can probably now go into if with init-statement.

Reply:
can't, it would still throw std::out_of_range if no key is found